core/src/main/resources-filtered/org/pentaho/platform/dataaccess/datasource/wizard/public/modeler.properties

```diff
@@ -1,5 +1,6 @@
 modeler.title=Data Source Model Editor
 ModelEditor.ERROR=Modeler Error
+modeler.warning=Modeler Warning
```

Comment:
Created extra property to handle message. Did not create other Internationalization property files ie German (modeler_de.properties), Japanese (modeler_jp.properties)... I was told it magically is created at the end of releases, if PM decide it is necessary.

Based the wording from related `ModelEditor.ERROR`.